gymnasium/wrappers/record_episode_statistics.py

```diff
@@ -96,8 +67,8 @@ def __init__(self, env: gym.Env, deque_size: int = 100):
     def reset(self, **kwargs):
         """Resets the environment using kwargs and resets the episode returns and lengths."""
         observations = super().reset(**kwargs)
-        self.episode_returns = np.zeros(self.num_envs, dtype=np.float32)
```

Comment:
Why this change? Pretty much all the floating point math in gymnasium works on float32, this change will either add an extra cast somewhere along the line, or require an explicit cast by user by breaking their code (numpy doesn't care, but pytorch does)

Reply:
Solely for passing the test for the VectorList wrapper, it checks each element with `assert isinstance(..., float)`, which only passed for float64 (not float32 or int). I agree I'd prefer to use float32, wasn't sure whether I could change pre-existing tests

Reply:
Can you point me to the specific test? File/line?

Reply:
https://github.com/Farama-Foundation/Gymnasium/blob/9154d2db4e3955a99d81d1b1d982349cb78c80e6/tests/wrappers/test_vector_list_info.py#L56

Reply:
Is it the case that `list_info[i]["episode"][stats]` used to be a list of floats, and now is a numpy array? Or am I misunderstanding something here?